### DISCUSSION TITLE
无反应问题

### DISCUSSION THREAD

**Discussion by zhongguanLiu** [lisai9093/YYS#6] [Q&A] [answered]
我使用管理员权限打开了IDE，我使用的YYS是官方桌面版，启动之后并未手动改变方框界面的大小；
例如当我输入4进入单人御魂模式，但是当我YYS游戏内进入准备单刷御魂界面时游戏没有任何的反映。
如果这个时候程序在运行，我打开存有图片的文件夹那么可以进行动作和响应的输出，也就是说这一版本的程序没有办法对YYS程序进行判断，是因为网易做了防止外界程序点击的功能还是我没有弄对操作？

**Comment by lisai9093** [comment]
我没有遇到过这个问题，刚刚测试了下也没有问题。确保在单刷界面：
![image](https://github.com/lisai9093/YYS/assets/3047171/ee616f76-8f16-45fb-b485-a0e82f62c435)

网易没有权限阻止这个脚本点击的，如果还是不行你可以试试手动截图。单刷对应的开始截图是"tiaozhan3.png"，你可以截图替换原文件试试。

另外一个可能的问题是要关闭显示的DPI功能，默认只支持100% DPI

**Comment by zhongguanLiu** [comment]
感谢回复。我重新截图后替换或者添加新的文件进去仍然没有反应（并且确保YYS内处于您给出的这个界面）。
对于DPI这个内容，我不是太了解，能否麻烦您给出一个详细的方法？（我直接启动的程序，并没有对窗口进行缩放）目前我的配置是这样的：
![image](https://github.com/lisai9093/YYS/assets/53946756/1c6e6da7-db57-4ab2-aa5b-4458889d9c2c)
![image](https://github.com/lisai9093/YYS/assets/53946756/794fd0f9-615b-40dc-8a30-8148cd76b0de)
![image](https://github.com/lisai9093/YYS/assets/53946756/b98f9f33-4aa0-42c3-8275-f716aa98a9ef)

**Reply by lisai9093** [reply]
DPI确实是100%没问题

另外一个可能性是要把YYS程序移动到左上角试试，脚本检测是默认左上角开始的

**Reply by zhongguanLiu** [reply]
移动到左上角这个方法是有效果的！
但是鼠标能够定位到挑战按钮却无法点击下去（有时候可以，没明白不能点击的原理）
并且在一次刷御魂结束后，也无法再次点击，是否是我替换后的判断还是不能触发点击按钮呢？
![image](https://github.com/lisai9093/YYS/assets/53946756/430e9863-ed2a-4f2d-aa1a-63bd7c73c811)
![image](https://github.com/lisai9093/YYS/assets/53946756/18adfaca-4eac-4964-b5a2-6f4e5a77b18f)

**Comment by zhongguanLiu** [comment, marked as answer]
非常感谢，这个问题已经彻底解决了。我没有用管理员权限打开pycharm!
感谢作者！
